File: index/src/test/resources/runtime-helpers/java-state/Entry.java
package probe;
public final class Entry {
  static final class Names { private final String value; Names(String value) { this.value = value; } String value() { return value; } }
  public static void main(String[] args) throws Exception { Class.forName(new Names(args[0]).value()).getDeclaredConstructor().newInstance(); }
}
class Used { public Used() { System.out.println("USED"); } }
class Unused { public Unused() { System.out.println("UNUSED"); } }

File: index/src/test/kotlin/dev/kartograph/index/fixture/helpercompanion/Entry.kt
package probe.helpercompanion
class Names { companion object { fun value(): String = "probe.helpercompanion.Used" } }
class Used { init { println("USED") } }
class Unused { init { println("UNUSED") } }
object Entry { @JvmStatic fun main(args: Array<String>) { Class.forName(Names.value()).getDeclaredConstructor().newInstance() } }
